replica/distributed_loader.cc

```diff
@@ -172,15 +172,12 @@ distributed_loader::make_sstables_available(sstables::sstable_directory& dir, sh
 
 future<>
 distributed_loader::process_upload_dir(distributed<replica::database>& db, sharded<db::view::view_builder>& vb, sstring ks, sstring cf) {
-    seastar::thread_attributes attr;
-    attr.sched_group = db.local().get_streaming_scheduling_group();
```

Comment:
I think we should keep this switch (using the sg we store onboard), so we are context independent. It also makes the two load paths uniform.

Reply:
I don't follow this suggestion, sorry :(

First, there's no "on board" sched group for this code. The distributed_loader's methods are all static, so is the process_upload_dir() one.

Next, if two load paths you mention are this code and sstabled_loader::load_and_stream, then keeping a switch here won't keep it uniform, because the latter doesn't switch at all

Reply:
Sorry, looks like I misunderstood this. Nevermind this comment.